apps/tenant-management-webapp-e2e/src/integration/calendar/calendar.page.ts

```diff
@@ -0,0 +1,89 @@
+class CalendarPage {
+
+  addCalendarOverviewTabBtn() {
+    return cy.get('[data-testid="overall-calendar-add-btn"]');
+  }
+  addCalendarBtn() {
+    return cy.get('[data-testid="add-calendar-btn"]');
+  }
+
+  addScriptModalTitle()
+  {
+    return cy.xpath('//goa-modal[@data-testid="add-calendar-modal" and @open="true"]//*[@slot="heading"]');
+  }
+
+  calendarModalCancelButton()
+  {
+    return cy.get('[data-testid="calendar-modal-cancel"]');
+  }
+
+  calendarModalSaveButton()
```

Comment:
Can this page object be shared with editCalendarModalSaveButton()? It's named calendarModalSaveButton(), and user would assume it's for any save button in any calendar modal.